examples/mithril-kitchen-sink/src/Welcome.js

```diff
@@ -0,0 +1,178 @@
+/** @jsx m */
+
+import m from 'mithril';
+import { linkTo, hrefTo } from '@storybook/addon-links';
```

Comment:
What I meant in my previous comment, is that we don't want our (demo) components to be dependant on any storybook addons. Think about the real world app. Probably you will expose some callbacks and will use them inside a `stories.js` file with bindings to the particular addon.